mavros/08cd181/08cd181.bug

```diff
@@ -25,13 +25,13 @@ bug:
   detected-by: developer
   reported-by: contributor
   issue: https://github.com/mavlink/mavros/issues/161
-  time-reported: 2014-11-02 (00:00)
-  reproducibility: N/A
+  time-reported: 2014-11-02T00:00:00+00:00
```

Comment:
```suggestion
  time-reported: 2014-11-02T13:28:17Z
```

@wasowski: if/when @ChrisTimperley agrees, you'll only have to accept the suggestion.

Reply:
Looks good. Thanks.